src/test/ui/suggestions/boxed-variant-field.stderr

```diff
@@ -6,14 +6,14 @@ LL |         Ty::List(elem) => foo(elem),
    |
    = note: expected enum `Ty`
             found struct `Box<Ty>`
-help: try dereferencing the `Box`
-   |
-LL |         Ty::List(elem) => foo(*elem),
-   |                               +
 help: try wrapping the expression in `Ty::List`
    |
 LL |         Ty::List(elem) => foo(Ty::List(elem)),
    |                               +++++++++    +
+help: consider unboxing the value
+   |
+LL |         Ty::List(elem) => foo(*elem),
+   |                               +
```

Comment:
Hmm, I really liked that the old suggestion was before the enum suggestion. The unbox suggestion is almost certainly correct, while the enum suggestion may be totally wrong. Can you try moving the box suggestion before the enum one again?